### PR TITLE
Various bug fixes & Chinese localization patch

### PR DESCRIPTION
**Pull request by Cyrillya**
Bug fix: #73, #67
Further Chinese localization and missing translation key added

Should we remove `GUIConfig`? Since it's no longer used

### CODE REVIEW

**Comment by Cyrillya** [comment]
image
What does this mean? Is there anything I should do?

File: Content/Tools/Editors/PlayerEditor.cs [hunk 1]
```diff
@@ -165,6 +166,9 @@ static string GetLocalizedText(string text)
 				return LocalizationHelper.GetText($"Tools.PlayerEditor.{text}");
 			}
 
+			if (BoundingBox.Contains(Main.MouseScreen.ToPoint()))
+				PlayerInput.LockVanillaMouseScroll("DragonLens: Item Editor");
```
Comment (ScalarVector1) [inline Content/Tools/Editors/PlayerEditor.cs:170]: Reason is incorrect, should be Player editor, not item editor

File: Localization/en-US.hjson
Comment (ScalarVector1) [inline Localization/en-US.hjson:1]: Any reason these were deleted? As far as I know these are required for the keybinds in the controls menu.

**Review by ScalarVector1 — COMMENTED** [review]
Few nits, main concern is the apparent deletion of keybind localization? Is there a reason these were removed that I missed?

**Comment by ScalarVector1** [comment]
> image What does this mean? Is there anything I should do?

It appears somehow the CI is freaking out, I'll investigate this

**Comment by ScalarVector1** [comment]
Warning: /home/runner/work/DragonLens/DragonLens/Content/GUI/Filters.cs(88,113): warning IDE2001: Embedded statements must be on their own line [/home/runner/work/DragonLens/DragonLens/DragonLens.csproj]
Warning: /home/runner/work/DragonLens/DragonLens/Content/GUI/Filters.cs(88,13): warning IDE0008: Use explicit type instead of 'var' [/home/runner/work/DragonLens/DragonLens/DragonLens.csproj]

Appears you have genuine conflicts with the formatting linter. Please resolve these issues.

File: Localization/en-US.hjson
Comment (Cyrillya) [reply Localization/en-US.hjson:1]: The keybinds are already handled in `Localization/Keybinds/*.hjson` in my previous pull request (#59). `.hjson` files under `Localization/` are actually obsolete and useless

File: Content/Tools/Editors/PlayerEditor.cs (hunk 1 above)
Comment (Cyrillya) [reply Content/Tools/Editors/PlayerEditor.cs]: Fixed ✅

**Comment by Cyrillya** [comment]
Issue solved ✅

File: Localization/en-US.hjson
Comment (ScalarVector1) [reply Localization/en-US.hjson:1]: got it, sounds good

**Review by ScalarVector1 — APPROVED** [review]

**Pull request merged**